.circleci/config.yml

```diff
@@ -1776,9 +1518,8 @@ workflows:
           context: SonarCloud
           filters:
             branches:
-              ignore:
-                - /.*-PERF/
-                - /.*-REGRESSION/
+              only:
+                - Disable-master
```

Comment:
We cannot just disable this workflow. We need the workflow, still, to build the project with running all tests for SonarCloud

Reply:
For this, I think we need enable the SonarCloud and codecov in GCP CI workflow as well.

Reply:
Agreed. I already mentioned this in our slack discussion as another option. If we want to completely turn off this workflow, the GCP CI workflow must use `build-artifact` step for full testings and SonarCloud instead of `build-platform-and-services
` step, as of right now, that skips tests when building the artifact.